recipes/cosmosis/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "cosmosis" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb36efee267a74eb74efb093bef13e27f845f4bacc2821bb4394dee482503446
+
+build:
+  number: 0
+  skip: True # [(py != 36) or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - liblapack
+    - libblas
+    - {{ mpi }}
```

Comment:
This section should only have the compilers IIRC.

Reply:
Same problem as above - I have left as-is for now.

Reply:
Something is wrong then. Let's make the change and see what is going on.